src/test/net/jpountz/lz4/LZ4FrameIOStreamTest.java

```diff
@@ -479,19 +463,26 @@ public void testUncompressableEnd() throws IOException {
     }
   }
 
-  private static boolean hasNativeLz4CLI() throws IOException, InterruptedException {
+  private static boolean hasNativeLz4CLI() {
+    boolean hasCli;
     try {
       ProcessBuilder checkBuilder = new ProcessBuilder().command("lz4", "-V").redirectErrorStream(true);
       Process checkProcess = checkBuilder.start();
-      return checkProcess.waitFor() == 0;
+      hasCli = checkProcess.waitFor() == 0;
     } catch (IOException | InterruptedException e) {
       // lz4 CLI not available or failed to execute; treat as unavailable to allow test skip
-      return false;
+      hasCli = false;
+    }
+
+    // Check if this is running in CI (env CI=true), see https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables
+    if (!hasCli && "true".equals(System.getenv("CI"))) {
+      Assert.fail("LZ4 CLI is not available, but should be for CI run");
     }
```

Comment:
Here is an example actions run which shows that this works as desired:
https://github.com/Marcono1234/lz4-java/actions/runs/19996530642/job/57344936259#step:6:2789

(for that run I had intentionally removed the LZ4 CLI from the Windows build)

Or would you prefer if the complete test class failed fast if the LZ4 CLI was not available (instead of the individual tests)?